docs/supported-technologies.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[supported-technologies]]
+== Supported technologies
+
+The Elastic APM Android agent is built on top of the https://opentelemetry.io[Open Telemetry Java SDK] -- extending its functionality while also automatically instrumenting various APIs and frameworks.
+This section lists all supported technologies.
+
+* <<supported-agp-versions>>
+* <<supported-languages>>
+* <<supported-ui-frameworks>>
+* <<supported-networking-frameworks>>
+
+[float]
+[[supported-agp-versions]]
+=== Android Gradle Plugin versions
+
+|===
+|Supported versions
+
+|7.2+, 7.3+
```

Comment:
Is this redundant? Or is it important to list both 7.2 and 7.3?

Reply:
Well, though I'm not sure if 7.2+ would be perceived as only the patch versions for 7.2, or if it also would cover further minor versions too (7.3, 7.4... etc). In general we're just planning to support every AGP version starting from 7.2.0 onwards (even major versions too), I'm fine to change this to a broader term that doesn't require us to list all the major/minor versions, I just don't know what could be the best approach though, maybe something like `>= 7.2.0`?

Reply:
I like `>= 7.2.0`. That's the syntax we use in the other agents.